Documentation/5.4.1/README.rst

```diff
@@ -1414,60 +1427,57 @@ The following parameters are mapped to values in the /etc/default/nuage-metadata
 
 
-Parameters Required for Ironic/Ironic-Inspector
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The following parameters are mapped to values in the /etc/nova/nova.conf on the Controller:
-
-::
-
-    NovaSchedulerDiscoverHostsInCellsInterval
-    Maps to discover_hosts_in_cells_interval parameter.
-
-    NovaSchedulerDefaultFilters
-    Maps to scheduler_default_filters parameter.
-
-
-The following parameters are mapped to values in the /etc/ironic/ironic.conf on the Controller:
-
-::
-
-    IronicCleaningDiskErase
-    Maps to cleaning_disk_erase parameter.
-
-    IronicIPXEEnabled
-    Maps to ipxe_enabled parameter.
-
-    IronicDhcpProvider
-    Maps to dhcp_provider parameter.
-
-
-Parameters Required for Ironic-Inspector
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The following parameter is mapped to value in the /etc/ironic-inspector/inspector.conf on the Controller:
-
-::
-
-    IronicInspectorExtraProcessingHooks
-    Maps to processing_hooks parameter.
-
-
-The following parameter is mapped to value in the /etc/ironic-inspector/dnsmasq.conf on the Controller:
-
-::
-
-    IronicInspectorIpRange
-    Maps to dhcp-range parameter.
-
-
-The following parameter is used to enable/disable ipxe on th Controller:
-
-::
-
-    IronicInspectorIPXEEnabled
-    Used to enable/disable ipxe
-
+.. .. Parameters Required for Ironic/Ironic-Inspector
+.. ..~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. ..
+.. ..The following parameters are mapped to values in the /etc/nova/nova.conf on the Controller:
+.. ..
+.. ..::
+.. ..
+.. ..    NovaSchedulerDiscoverHostsInCellsInterval
+.. ..    Maps to discover_hosts_in_cells_interval parameter.
+.. ..
+.. ..    NovaSchedulerDefaultFilters
+.. ..    Maps to scheduler_default_filters parameter.
+.. ..
+.. ..
+.. ..The following parameters are mapped to values in the /etc/ironic/ironic.conf on the Controller:
+.. ..
+.. ..::
+.. ..
+.. ..    IronicCleaningDiskErase
+.. ..    Maps to cleaning_disk_erase parameter.
+.. ..
+.. ..    IronicIPXEEnabled
+.. ..    Maps to ipxe_enabled parameter.
+.. ..
+.. ..
+.. ..Parameters Required for Ironic-Inspector
+.. ..~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. ..
+.. ..The following parameter is mapped to value in the /etc/ironic-inspector/inspector.conf on the Controller:
+.. ..
+.. ..::
+.. ..
+.. ..    IronicInspectorExtraProcessingHooks
+.. ..    Maps to processing_hooks parameter.
+.. ..
+.. ..
+.. ..The following parameter is mapped to value in the /etc/ironic-inspector/dnsmasq.conf on the Controller:
+.. ..
+.. ..::
+.. ..
+.. ..    IronicInspectorIpRange
+.. ..    Maps to dhcp-range parameter.
+.. ..
+.. ..
+.. ..The following parameter is used to enable/disable ipxe on th Controller:
+.. ..
+.. ..::
+.. ..
+.. ..    IronicInspectorIPXEEnabled
+.. ..    Used to enable/disable ipxe
+.. ..
 
 Parameters Required for VRS Offload
```

Comment:
Can you please also talk about 
  ESWITCH_IPV4_TTL_MODIFY_ENABLE: True
  PRIO_TAG_REQUIRED_EN: True
parameters ?